src/Microsoft.ML.FastTree/Training/EnsembleCompression/LassoBasedEnsembleCompressor.cs

```diff
@@ -450,7 +450,7 @@ private LassoFit GetLassoFit(IChannel ch, int maxAllowedFeaturesPerModel)
             // First lambda was infinity; fixing it
             fit.Lambdas[0] = Math.Exp(2 * Math.Log(fit.Lambdas[1]) - Math.Log(fit.Lambdas[2]));
 
-            TimeSpan duration = DateTime.Now - startTime;
+            TimeSpan duration = DateTime.UtcNow - startTime;
             ch.Info("Elapsed time for compression: {0}", duration);
```

Comment:
This in particular is a bit weird and doesn't really have an effect I'd think.

Honestly this code here ought to have been using stopwatch.

Reply:
You're right, this has no effect. I have added a stopwatch here instead as per your suggestion